### DISCUSSION TITLE
An archive list of what has already downloaded

### DISCUSSION THREAD

**Discussion by Proxibyte** [UltimaHoarder/UltimaScraper#1398] [Ideas]
A feature like that youtube-dl does, makes a list of files already downloaded and checks against the list so you can move the downloaded files out and not worry about it downloading them again. could make a list for each account to make it cleaner, list off file names and check if they are already downloaded.

Might be useful for people like me who like storing the files with a proper name not the base name most users have.

**Comment by UltimaHoarder** [comment]
It already can do this. Just save the metadata files in a different directory other than the model folder
